libc.in

```diff
@@ -94,6 +94,8 @@ elif [ -d "$SYSCONFDIR"/resolvconf ]; then
 	fi
 fi
 : ${resolv_conf:=/etc/resolv.conf}
+# TODO: use dirname "$resolv_conf"
+: ${resolv_conf_tmp:="/etc/.resolv.conf.openresolv"}
```

Comment:
You could write this rather than using a function:

`${resolv_conf_tmp:=${resolv_conf}.tmp}`

Reply:
I wanted to hide temporary file like Network Manager does. But okay, it is not needed and would make it simpler. Added $$ to temporary file to prevent possible race conditions.